cookbook/email/email.rst

```diff
@@ -138,6 +138,11 @@ template might look something like this:
     {# Makes an absolute URL to the /images/logo.png file #}
     <img src="{{ absolute_url(asset('images/logo.png')) }}"
 
+.. versionadded:: 2.7
+    Support for creating absolute URLs with the ``asset()`` function will
+    be dropped in Symfony 3.0. Starting with Symfony 2.7, you should use
+    the ``absolute_url()`` function instead.
```

Comment:
I think we usually write something like this:

```rst
The ``absolute_url()`` function was introduced in Symfony 2.7. Prior to
2.7, the ``asset()`` function has an argument to enable returning an
absolute url.
```

It feels a little weird to talk about a feature that is removed, without actually using that function in the doc.